File: Hahn.ApplicatonProcess.Application/Hahn.ApplicatonProcess.July2021.Domain/Model/Asset.cs
using Hahn.ApplicatonProcess.July2021.Domain.Model;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace Hahn.ApplicatonProcess.July2021.Domain
{
    [Table("Asset")]
    public class Asset
    {
        [Key]
        public string assetId { get; set; }
        public string Id { get; set; }
        public int Rank { get; set; }
        public string Symbol { get; set; }
        public string Name { get; set; }
        public double Supply { get; set; }
        public double? MaxSupply { get; set; }
        public double MarketCapUsd { get; set; }
        public double VolumeUsd24Hr { get; set; }
        public double PriceUsd { get; set; }
        public double ChangePercent24Hr { get; set; }
        public double? Vwap24Hr { get; set; }
        public string Explorer { get; set; }
        public User User { get; set; }
    }
}

File: hahn-application/src/entities/Asset.ts
import { User } from "./User";

export class Asset {
  public id: string;
  public rank: string;
  public symbol: string;
  public name: string;
  public supply: string;
  public maxSupply: string;
  public marketCapUsd: string;
  public volumeUsd24Hr: string;
  public priceUsd: string;
  public changePercent24Hr: string;
  public vwap24Hr: string;
  public explorer: string;
  public user: User;
}
